sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/AnalyzeSentimentAsyncClient.java

```diff
@@ -50,57 +50,110 @@
         this.service = service;
     }
 
-    Mono<Response<DocumentResultCollection<AnalyzeSentimentResult>>> analyzeSentimentBatchWithResponse(
+    /**
+     * Helper function that analyzes a batch of documents and returns {@link TextAnalyticsPagedFlux} that is a paged
+     * flux contains {@link AnalyzeSentimentResult}.
+     *
+     * @param textInputs A batch of input texts.
+     * @param options The request options, such as the training model version and to show statistics.
```

Comment:
should we include a link to `TextAnalyticsRequestOptions ` instead of listing the properties?